tests/cypress/e2e/helpers.ts

```diff
@@ -3,9 +3,10 @@ export const inputGraphTypeSelector = 'input[id="input_Graph type"]';
 export const inputHostLabelsSelector = 'input[id="react-select-15-input"]';
 export const inputHostSelector = 'input[id="input_Hostname"]';
 export const inputMetricSelector = 'input[id="input_Metric"]';
+export const inputRegexSelector = 'input[class="css-1mlczho-input-input"]'; //TODO: introduce id for this input selector.
```

Comment:
You might want to take a look at [this commit](https://github.com/tribe29/grafana-checkmk-datasource/pull/126/commits/893bdffd45371431b153998582d8fd8d38eb295a) from the other pull request. It's an example of how to put test Ids onto elements we have control over.

Reply:
Ack. Thanks for the fix :)